src/Components/Alert/Components/Steps/Details.tsx

```diff
@@ -45,73 +47,89 @@ export const Details: FC = () => {
         }
 
         return (
-          <Flex flexDirection="column" p={2}>
-            <Text variant="lg">Create Alert</Text>
-            <Spacer y={2} />
-            <AlertNameInput />
-            <Spacer y={4} />
-            <Join separator={<Spacer y={4} />}>
-              <Box>
-                <Text variant="sm-display" mb={1}>
-                  Filters
-                </Text>
-                <Flex flexWrap="wrap" gap={1}>
-                  <CriteriaPills />
-                </Flex>
-              </Box>
+          <Box
+            maxWidth={[null, 500]}
+            style={{
+              ...(isMounted
+                ? {
+                    opacity: 1,
+                    transform: "translateX(0)",
```

Comment:
@tam-kis - lets not add animation here if we can avoid it. Having a screen that is snappy and fast is much better than adding transitions for the sake of transitions. If we're going to add UI that transitions between screens, this is something that should be raised up to the design-system working group and ideally be implemented by @dzucconi.

Reply:
This is perfectly fine to merge without the animation -- get the functionality in, then we can work it in in a more appropriate fashion later, via #design-system and ideally a ticket in Diamond.

Reply:
I totally don't have a problem with this transition actually :) — do I wish we had a consistent way of doing it, sure, but this is simple enough and helps communicate the UI change.

Reply:
Cool, if yr ok with it then 👍 Just wanted to be sure that things are flowing through the proper channels. Will resolve!